MaxMind.MinFraud/Request/CreditCard.cs

```diff
@@ -154,14 +163,21 @@ public sealed class CreditCard
             }
         }
 
+        /// <summary>
+        /// Whether or not the 3DS check was successful, as provided by the end
```

Comment:
Sorry, I should've included this `<summary>` in first review comment re: reasons not to include this value